### ISSUE TITLE
Get a site resource by path doesn't work for Germany environment

### ISSUE THREAD

**Issue by romanSlivinskyi** [microsoftgraph/microsoft-graph-docs#3745]
Using 

`GET /sites/{hostname}:/{relative-path}`

 to retrieve properties for a site resource in **_Germany environment_** returns the following error: 
```
{
  "error": {
    "code": "invalidRequest",
    "message": "Provided id has an invalid hostname for this tenancy",
    "innerError": {
      "request-id": "***",
      "date": "2019-01-09T12:29:08"
    }
  }
}
```
The request was to: 

`https://graph.microsoft.de/v1.0/sites/{host}.sharepoint.de:/{relative-path}`

**Comment by CodedBeard** [comment]
We are now seeing this exact same issue on normal Office365. We have an api that looks up site ids via relative path, that has been working for months, and all of a sudden we have started receiving this same error.
We've even found that the graph explorer is showing the same error for queries that show in my query history they previously worked, but now return this error. If we search for the site instead, it does return the information which shows that the hostname being provided to the relative query is in fact correct as we would expect.

**Comment by mgreasly** [comment]
Yes, we have noticed this in our Singapore tenant since 13/01 as well. Has anyone seen any workarounds or responses from Microsoft?

**Comment by mgreasly** [comment]
The error message seems to be coming directly from the SharePoint Rest API call that is (presumably) underlying this graph call.

**Comment by rahzel44** [comment]
I doesn't work also in France since a couple of days ! For now, I haven't seen any Microsoft responses to fix that issue...

**Comment by kimrohdin** [comment]
Same in our environment since 13/01

**Comment by bredonkulous** [comment]
I have been seeing this on and off for the past week. Now this morning it's 100% consistent across both of my tenants and through different networks. We have a script using this and now i get only 400 errors back for only sites under the /sites/ managed path. We also have sites under /portals/ that DO work with this command. Same experience using the graph explorer. Located in the Central US.

**Comment by npshaheen** [comment]
Same issue--saw this intermittently this past Friday, now the error occurs every time when trying to retrieve a site list. Here is a sample (with actual values removed)--note that this has worked fine for more than a year:
https://graph.microsoft.com/v1.0/sites/mysite.sharepoint.com:/sites/abc:/lists/88888888-8888-8888-8888-888888888888/items?expand=fields

**Comment by xstos** [comment]
Same thing appears to be happening to us. We have a Canadian Office 365 subscription and intermittent failures using this endpoint:

https://graph.microsoft.com:443/v1.0/sites/contoso.sharepoint.com:/sites/Dev:

With the same error "Provided id has an invalid hostname for this tenancy"

**Comment by CodedBeard** [comment]
Looks like this has been resolved, at least for us it is now working again as before. Shame this thread never got a response from MS, and our attempts at talking to support were equally as fruitless.

**Comment by mgreasly** [comment]
Yes, mysteriously started working in our tenants yesterday. No response from the premier support ticket raised by clients though. I'll update if we get a response with any useful information..

**Comment by westleyMS** [comment]
Sorry we didn't respond on this thread, we did identify the issue when it was reported at the beginning and were working to resolve from day 1. The fix that was rolled out should be on nearly all tenants now.

**Comment by tknaller** [comment]
i'm having the very same issue today


`GET https://graph.microsoft.com:443/v1.0/sites/mytenant.sharepoint.com`

```json

{
    "error": {
        "code": "invalidRequest",
        "message": "Invalid hostname for this tenancy",
        "innerError": {
            "request-id": "014ab68b-4d6b-4582-a91e-49b7b66de0b7",
            "date": "2020-04-09T06:29:08"
        }
    }
}
```
